runtime/typescript/src/dfa/DFASerializer.ts

```diff
@@ -59,12 +59,17 @@ export class DFASerializer {
 			states.sort((o1, o2) => o1.stateNumber - o2.stateNumber);
 
 			for (const s of states) {
-				const edges: Map<number, DFAState> = s.getEdgeMap();
+				const edges: Array<DFAState> = s.getEdgeMap();
```

Comment:
💭 Why `Array<DFAState>` instead of `DFAState[]`? I'm sure I used to know the answer...

Reply:
They are the same thing.   It just was easier to substitute.

Reply:
Nice to hear from you.

Reply:
P.S.  most of these changes are from earlier PR #454.